src/model/source-range.ts

```diff
@@ -12,13 +12,27 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+/**
+ * Describes a range of text within a source file.
+ *
+ * NOTE: See `Position` documentation below for an important information on
```

Comment:
What about something a bit shorter and sweeter. What about:

`
Note: columns and line positions are indexed from zero. Consider displaying them to users as one-indexed to match text editor convention.
`

And then dropping the note from Position.